packages/test-support/package.json

```diff
@@ -45,20 +45,27 @@
     "test:unit": "mocha \"./test/unit/**/*.spec.js\""
   },
   "dependencies": {
-    "@appium/support": "file:../support",
     "@babel/runtime": "7.17.9",
     "bluebird": "3.7.2",
+    "get-port": "5.1.1",
```

Comment:
shell we also review base driver's deps and remove obsolete stuff?

Reply:
I'm not sure exactly what you mean, but neither `@appium/base-driver` nor `@appium/base-plugin` have peer dependencies on Appium, since that introduces a cyclic dependency.  Appium depends on both of these deps directly.

Reply:
I mean we could probably remove some unused dependencies from the base driver after this code has been extracted from it

Reply:
ah, hmm, maybe.  I'll take a look, but I suspect all that stuff wound up in the root devDependencies

Reply:
yeah I'm not seeing anything in there that jumps out at me

```json
  "dependencies": {
    "@appium/support": "file:../support",
    "@babel/runtime": "7.17.9",
    "@colors/colors": "1.5.0",
    "async-lock": "1.3.1",
    "asyncbox": "2.9.2",
    "axios": "0.27.2",
    "bluebird": "3.7.2",
    "body-parser": "1.20.0",
    "es6-error": "4.1.1",
    "express": "4.18.1",
    "http-status-codes": "2.2.0",
    "lodash": "4.17.21",
    "lru-cache": "7.8.1",
    "method-override": "3.0.0",
    "morgan": "1.10.0",
    "serve-favicon": "2.5.0",
    "source-map-support": "0.5.21",
    "validate.js": "0.13.1"
  }
```

Reply:
wait, maybe axios?

Reply:
nope, it's used for the proxy stuff.